docs/DeveloperGuide.md

```diff
@@ -2,21 +2,133 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links 
+to the original source as well}
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+The following is our class diagram for our whole project.
 
+![Local Image](UML/CLiRental.png)
+
+### Sequence diagrams
+
+1. `Adding a customer`
+
+The following sequence diagram will explain the sequence of events after the user inputs an add-user command.
+
+![Local Image](UML/AddCustomerSequence.png)
+
+2. `Creating Car file and loading file`
```

Comment:
![image](https://github.com/user-attachments/assets/3f3b3f9b-2ab7-4a93-b074-a1af99185227)

Seems like there was an error when rendering the document, it should be 2. instead of 1.